README.md

```diff
@@ -1,65 +1,60 @@
-* charter: [https://www.w3.org/2017/vc/WG/charter.html](https://www.w3.org/2017/vc/WG/charter.html)
+* Charter: [https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html](https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html)
 * Chairs
   * Brent Zundel - @brentzundel
-  * Wayne Chang - @wyc
+  * Kristina Yasuda - @Sakurann
 
 ### Verifiable Credentials github repos
-* [Data Model](https://github.com/w3c/vc-data-model)
 * [Use Cases](https://github.com/w3c/vc-use-cases)
+* [Data Model](https://github.com/w3c/vc-data-model)
+* [Implementation Guide](https://github.com/w3c/vc-imp-guide/)
 * [Data Model Test Cases](https://github.com/w3c/vc-test-suite)
 
-### Other useful links
-* [Public group email archive](https://lists.w3.org/Archives/Public/public-vc-wg/)
+### Discussion Forums
+* [W3C Credentials Community Group Mailing List (Incubation)](https://lists.w3.org/Archives/Public/public-credentials/)
+* [W3C Verifiable Credentials WG Mailing List (Standardization)](https://lists.w3.org/Archives/Public/public-vc-wg/)
 
 ## Process Overview for VC Data Model Pull Requests
-1. For now, we will focus only on merging new errata PRs into this repository,
-   but encourage activity related to new features.
-2. Once a PR is opened, chairs and editors make judgement call on whether
-   changes are substantive or editorial.
+1. Anyone can open a PR on the repository. Note that for the PR to be merged,
+   the individual must agree to the
+   [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy/).
+2. Once a PR is opened on the `v2.0` branch, chairs and editors make judgement
+   call on whether changes are substantive or editorial.
    <dl>
      <dt>Editorial</dt>
-     <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
+     <dd>Mark with "editorial" tag</dd>
      <dt>Substantive</dt>
-     <dd>Mark with "substantial" tag. Bug fixes are merged into separate branch "v1.2". New Features stay around as an open PR.</dd>
+     <dd>Mark with "substantive" tag.</dd>
    </dl>
-3. W3C CCG is notified of PRs that will be merged in the next 14 days if there
-   are no objections.
-4. When it's determined a new recommendation should go out, the W3C Verifiable
-   Credentials Working Group members meet, review all the PRs that have been
-   merged, and make a formal recommendation if agreement is reached.
-
-### Roadmap for 2021
-- one editorial update (v1.1?)
-- one substantive update (v1.2?)
-
-- VC Test Suite Refactoring
-- Start planning VC v2 Work, request a rechartering 3-6 months before end of
-  year to keep VC WG functioning.
-
-### Focus areas
-- [v1] Fixing a specific bug
-- [v1] Update examples in the spec to make them modern
-- [v2] VC `@context` needs updating, possibly with security vocab modularized
-  into smaller components instead of all included into a large context file.
+3. W3C CCG is automatically notified of PRs as they are raised and discussed.
```

Comment:
Has an automatic procedure been set up for this?

Reply:
Yep, via [github-notify-ml](https://github.com/w3c/github-notify-ml-config): 

CCG receives weekly updates for all VCWG specs on Mondays: https://lists.w3.org/Archives/Public/public-credentials/2022Jul/0104.html